monkey/common/agent_configuration/validators/ip_ranges.py

```diff
@@ -0,0 +1,24 @@
+import re
+
+from marshmallow import ValidationError
+
+ip_regex = r"((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)"
```

Comment:
Maybe there's something in https://docs.python.org/3/library/ipaddress.html we can use instead.

It might also help to use marshmallow's[ IP field](https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html#marshmallow.fields.IP).

Reply:
For CIDR, we can use [IPv4Network](https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Network)

Reply:
> For CIDR, we can use [IPv4Network](https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Network)

We don't support network masks, which IPv4Network supports

Reply:
We can try to cast to IP, using python IPv4Address. Marshmallow IP field doesn't match, because this is not an IP field, it's a string with one of 4 structures

Reply:
#2135